### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by androidacy-user** [kelseyhightower/nocode#4581]
Hi i'm trying to build my application. The code is thus:
```
```
Trying the following command to build:
```
```
Gives the following error:
```
```

Is this a bug or an I doing something wrong.

**Comment by Richrdo** [comment]
是的，从你的这行代码：
`

`
可以看出有语法错误，你首先需要：
`

`
然后将它：
`

`
接着，你能得到以下结果
`

`

**Comment by luizppa** [comment]
I believe that is intended behaviour. Have you tried the following?
```

```

**Comment by androidacy-user** [comment]
> I believe that is intended behaviour. Have you tried the following?
> 
> ```
> 
> ```

Ah, i forgot to add
```
```
To the build command as flags. Let me try again.

**Comment by fxtressia** [comment]
Did it work? I'm getting the same issue.

My code:
```

```

<details>
<summary>Log file (long)</summary>

```

```


</details>

**Comment by lingtian1211** [comment]
？？？？

**Comment by androidacy-user** [comment]
> Did it work? I'm getting the same issue.
> 
> My code:
> 
> ```
> 
> ```
> 
> Log file (long)
> ```
> 
> ```

You need to update your compiler to `        ` version first and add `          ` to compilation flags

**Comment by mading187** [comment]
I am a superCoder, i want to help you try my best
### First
```
   
```
### Next
```
   
```
### Finally
```
   
```
look , it's work
you are welcome
